packages/graphql/src/mutations/_txHelper.js

```diff
@@ -311,6 +311,11 @@ async function handleCallbacks({ callbacks, val }) {
 async function handleHash({ hash, from, mutation }) {
   debug(`got hash ${hash}`)
 
+  if (typeof hash !== 'string' || ![66, 64].includes(hash.length)) {
```

Comment:
super nit: since this is used in a couple places, maybe defined a isTxHashValid method.

Reply:
Agreed.  I think it just slowly escalated and wasn't supposed to be used in so many places originally.  Will do, thanks